### PR TITLE
Feature/specify entry chunks

### LINKED ISSUE #3: Specify entry?
can Specify entry?   my entry is mutil.   every entry injected,  that is bad

### PR DESCRIPTION
**Pull request by Demivan**
I needed functionality to specify entry chunks in which to inject CSS mapping, this pull request implements it.
This should close #3

### CODE REVIEW

File: src/flush-css-chunks-webpack-plugin.js [hunk 1]
```diff
@@ -64,6 +65,10 @@ class FlushCSSChunks {
             return;
           }
 
+          if (this.options.entryOnly && this.options.entries && this.options.entries.every(entry => chunk.name !== entry)) {
```
Comment (mario-jerkovic) [inline src/flush-css-chunks-webpack-plugin.js]: Maybe we should check here if the `entries` is array like `Array.isArray(this.options.entries)` and replace the `this.options.every(...)` with `!this.options.entries.includes(chunk.name)`

**Review by mario-jerkovic — CHANGES_REQUESTED** [review]

File: src/flush-css-chunks-webpack-plugin.js (hunk 1 above)
Comment (Demivan) [reply src/flush-css-chunks-webpack-plugin.js]: Will change to ```includes```.

What should we do if it is not an array? Throw error or just ignore it?

Comment (mario-jerkovic) [reply src/flush-css-chunks-webpack-plugin.js]: I think we should throw and Error which "describes" wrong configuration/option property

Comment (Demivan) [reply src/flush-css-chunks-webpack-plugin.js]: Switched to `includes` and added Error about wrong options

**Review by mario-jerkovic — APPROVED** [review]

**Comment by mario-jerkovic** [comment]
@Demivan Thank you for your time :). I owe you a beer 🍻

**Pull request merged**